src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffendersResourceTest.java

```diff
@@ -121,6 +121,40 @@ public void getFullOffenderInformation() {
         assertThatJsonFileAndStatus(response, 200, "offender_detail.json");
     }
 
+    @Test
+    public void testOffenderWithActiveRecallOffence() {
+        final var token = authTokenHelper.getToken(AuthTokenHelper.AuthToken.GLOBAL_SEARCH);
+
+        final var httpEntity = createHttpEntity(token, null);
+
+        final var response = testRestTemplate.exchange(
+                "/api/offenders/{nomsId}",
+                HttpMethod.GET,
+                httpEntity,
+                new ParameterizedTypeReference<String>() {
+                },
+                "A1234AC");
+
+        assertThatJsonFileAndStatus(response, 200, "offender_detail_recall.json");
+    }
```

Comment:
going forward it might be worth thinking about bring the json asserts inline rather than referring to a file.